sdk/core/azure-core-http-netty/src/main/java/com/azure/core/http/netty/NettyAsyncHttpClient.java

```diff
@@ -153,4 +153,21 @@
             }
         };
     }
+
+    private static Mono<byte[]> collectResponseWithHint(Flux<ByteBuffer> data, String contentLengthHeader) {
+        if (contentLengthHeader == null) {
+            return FluxUtil.collectBytesInByteBufferStream(data);
+        } else {
+            try {
+                int contentLength = Integer.parseInt(contentLengthHeader);
+                if (contentLength > 0) {
+                    return FluxUtil.collectBytesInByteBufferStream(data, contentLength);
+                } else {
+                    return Mono.just(new byte[0]);
+                }
+            } catch (NumberFormatException ex) {
+                return FluxUtil.collectBytesInByteBufferStream(data);
+            }
+        }
+    }
```

Comment:
This code is very similar to the one in JdkAsyncHttpClient. Can this static method be made into a utility method in core?

Reply:
Thoughts on `FluxUtil.collectBytesInNetworkResponse(Flux<ByteBuffer>, HttpHeaders)`?

Reply:
`FluxUtil.collectBytesFromNetworkResponse(Flux<ByteBuffer>, HttpHeaders)`